<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlowneOkno</class>
 <widget class="QMainWindow" name="GlowneOkno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GlowneOkno</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>651</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>651</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>204</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nr &quot;tury&quot;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>StanGry</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_StanGry">
       <property name="text">
        <string>Wybor warunkow</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcdNumber_nrTury">
       <property name="intValue" stdset="0">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_gracze">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>121</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Ilosc Graczy</string>
    </property>
    <widget class="QRadioButton" name="radioButton_1gracz">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>1 gracz</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2graczy">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>2 graczy</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QDial" name="dial_szybkoscGry">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>30</number>
    </property>
   </widget>
   <widget class="QDial" name="dial_zlotoPoczatkowe">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="singleStep">
     <number>50</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_zlotoPoczatkowe">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="intValue" stdset="0">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_szybkoscGry">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="intValue" stdset="0">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zloto poczatkowe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Szybkosc gry</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_akcja">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>240</y>
      <width>172</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Utworz serwer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gracz1">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gracz 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gracz2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gracz 2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
